<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1171</width>
    <height>808</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">SnsTradeBot</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>631</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>조건식</string>
    </property>
    <widget class="QPushButton" name="btn_refresh_condition">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>refresh</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="table_condition">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>541</width>
       <height>141</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>11</number>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
    <widget class="QTableWidget" name="table_temp_stock">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>241</width>
       <height>121</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>5</number>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column/>
     <column/>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>233</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>조건 검색 결과</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_temp_code_add">
        <property name="text">
         <string>대상 종목에 추가</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>1011</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>보유/관심종목</string>
    </property>
    <widget class="QTableWidget" name="table_balance">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>651</width>
       <height>241</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="rowCount">
      <number>10</number>
     </property>
     <property name="columnCount">
      <number>10</number>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>181</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,1">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="QLineEdit" name="edit_code">
        <property name="placeholderText">
         <string>종목코드</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_code_add">
        <property name="text">
         <string>추가</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>297</x>
       <y>270</y>
       <width>361</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="btn_interest_balance">
        <property name="text">
         <string>미보유 종목 조회</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_real">
        <property name="text">
         <string>전체 실시간 등록</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>20</y>
       <width>311</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QComboBox" name="combo_buy"/>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>매도전략</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="btn_buy_strategy_clear">
        <property name="text">
         <string>선택종목 매수전략 clear</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="combo_sell"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>매수전략</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="btn_sell_strategy_add">
        <property name="text">
         <string>선택종목에 매도전략 추가</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="btn_buy_strategy_add">
        <property name="text">
         <string>선택종목에 매수전략 추가</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="btn_sell_strategy_clear">
        <property name="text">
         <string>선택종목 매도전략 clear</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="midLineWidth">
           <number>0</number>
          </property>
          <property name="text">
           <string>param: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_buy_param"/>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>param: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_sell_param"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>System</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>295</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="btn_load">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_save">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_print">
        <property name="text">
         <string>Print</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>331</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,0">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>계좌번호</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_account">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_balance">
       <property name="text">
        <string>보유 종목 조회</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>combo_buy</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_combo_buy_strategy_changed(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>746</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>823</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>combo_sell</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_combo_sell_strategy_changed(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>910</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>919</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_buy_strategy_add</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_btn_buy_strategy_add_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>778</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>835</x>
     <y>569</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_sell_strategy_add</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_btn_sell_strategy_add_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>945</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>960</x>
     <y>527</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_buy_strategy_clear</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_btn_buy_strategy_clear_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>767</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>831</x>
     <y>592</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_sell_strategy_clear</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_btn_sell_strategy_clear_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>958</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>972</x>
     <y>491</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>table_balance</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_table_balance_selection_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>table_balance</sender>
   <signal>itemChanged(QTableWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_table_balance_item_changed(QTableWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_combo_buy_strategy_changed(QString)</slot>
  <slot>on_combo_sell_strategy_changed(QString)</slot>
  <slot>on_btn_buy_strategy_add_clicked()</slot>
  <slot>on_btn_sell_strategy_add_clicked()</slot>
  <slot>on_btn_buy_strategy_clear_clicked()</slot>
  <slot>on_btn_sell_strategy_clear_clicked()</slot>
  <slot>on_table_balance_selection_changed()</slot>
  <slot>on_table_balance_item_changed(QTableWidgetItem*)</slot>
 </slots>
</ui>
